tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -168,19 +168,19 @@ stratovirt-tarball:
 rootfs-image-tarball: agent-tarball
 	${MAKE} $@-build
 
-rootfs-image-confidential-tarball: agent-opa-tarball coco-guest-components-tarball kernel-confidential-tarball
+rootfs-image-confidential-tarball: agent-opa-tarball pause-image-tarball coco-guest-components-tarball kernel-confidential-tarball
 	${MAKE} $@-build
 
-rootfs-image-tdx-tarball: agent-opa-tarball kernel-confidential-tarball
+rootfs-image-tdx-tarball: agent-opa-tarball pause-image-tarball kernel-confidential-tarball
 	${MAKE} $@-build
```

Comment:
@fidencio - can we remove the tdx and sev targets now that they are (IIRC) using the generic confidential image?

Reply:
In the same PR, but in a different commit, please, remove the **kernel**, **initrd**, and **image** related to tdx / sev / snp.

I thought I had that covered as part of https://github.com/kata-containers/kata-containers/pull/9026, but I missed those bits. :-/

Reply:
Ok, let's remove them.

Reply:
Done.